code/style.css

```diff
@@ -1,3 +1,129 @@
 body {
-  background: #FFECE9;
-}
\ No newline at end of file
+  background: white;
+  margin: 0;
+  font-family: Helvetica, sans-serif;
+  color: #6a727b;
+  font-weight: 100;
+}
+
+header {
+  background-color: #24292f;
+  margin-bottom: 20px;
+  text-align: center;
+}
+
+#hamburger-icon {
```

Comment:
I think it´s better practice to use classes for styling and save the ids for JavaScript, due to specificity :)